host/src/config.rs

```diff
@@ -70,16 +70,19 @@ pub const L2CAP_RX_QUEUE_SIZE: usize = raw::L2CAP_RX_QUEUE_SIZE;
 /// Ensuring fair access to the pool is done configuring the QoS policy when creating
 /// the host resources.
 ///
-/// Default: 1.
+/// Default: 8.
 pub const L2CAP_RX_PACKET_POOL_SIZE: usize = raw::L2CAP_RX_PACKET_POOL_SIZE;
 
-/// GATT packet pool size.
+/// L2CAP TX packet pool size
+///
+/// This is the tx packet pool size of every l2cap channel. There has to be at least
```

Comment:
I deliberately didn't mention all, because the l2cap channels does not use the TX packet pool (yet), only the ATT (channel 4) channel, they send ACL packets directly to the controller.

I want to do some more testing with/without buffering the outbound TX in the host, because it has an important impact on code size and performance.

For GATT, I think it improves usability that we don't need to pass the 'stack'  and the controller generic around everywhere for sending.